src/Concerns/TransformableData.php

```diff
@@ -8,7 +8,7 @@ trait TransformableData
 {
     public function all(): array
     {
-        return $this->transform(transformValues: false);
+        return $this->transform(transformValues: false, mapPropertyNames: false);
```

Comment:
Let's keep the functionality of all as is for now since it would be a breaking change!

Reply:
mapPropertyNames with a false value

was already removed from the function

Reply:
Shouldn't `mapPropertyNames: true` be set here if we want to keep the functionality of all as is?

Reply:
> Shouldn't `mapPropertyNames: true` be set here if we want to keep the functionality of all as is?

yes it is. transform function has the mapPropertyNames true by default. 
https://github.com/Nir-An/laravel-data/blob/0a617c65a4bbf609b513bf457b3297d174d9b7f4/src/Concerns/BaseData.php#L100